_docs/reference/expression-language.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Mixer Config Expression Language
+overview: Mixer config expression language reference.
+
+order: 43
+
+layout: docs
+type: markdown
+---
+
+{% capture mixerConfig %}{{home}}/docs/concepts/policy-and-control/mixer-config.html{% endcapture %}
+
+This page describes how to use the Mixer config expression language (CEXL).
+
+## Background
+
+Mixer configuration uses an expression language (CEXL) to specify [selectors]({{mixerConfig}}#selectors) and [mapping expressions]({{mixerConfig}}#attribute-expressions). CEXL expressions map a set of typed [attributes]({{home}}/docs/concepts/policy-and-control/attributes.html) and constants to a typed [value](https://github.com/istio/api/blob/master/mixer/v1/config/descriptor/value_type.proto#L23).
+  
+```
+f(attr0, attr1, ..., constant0, constant1, ...) --> value
```

Comment:
This seems a bit too 'programmy' for an operator-level presentation, I'd just remove this code sample.